src/main/java/com/spotinst/sdkjava/model/ElastigroupItfLoadBalancer.java

```diff
@@ -12,6 +12,7 @@ public class ElastigroupItfLoadBalancer {
     private Set<String>                   isSet;
     private String                        loadBalancerArn;
     private List<ElastigroupListenerRule> listenerRules;
+    private DefaultStaticTargetGroups     defaultStaticTargetGroups;
```

Comment:
please change this to list<> as it is array of objects

Reply:
resolved